app/views/layouts/_header.html.erb

```diff
@@ -49,6 +49,7 @@
             <li><a href="/wiki/"><%= t('layout._header.projects.research.publiclab_wiki') %></a></li>
             <li><a href="/tools"><%= t('layout._header.projects.research.tools') %></a></li>
             <li><a href="/places"><%= t('layout._header.projects.research.places') %></a></li>
+            <li><a href="/users/map"><%= t('layout._header.projects.research.users_map') %></a></li>
```

Comment:
Here, the link would be `/users/map` -- @ebarry - can you suggest a better URL for a map of members? maybe `/people/map` or `/map/people`?